packages/dcos-launch/extra/launch/ip-detect/aws_public.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+set -o nounset -o errexit
+
```

Comment:
Yeah, this was copy/paste but we should take this moment to add some consistency